encoding.py

```diff
@@ -0,0 +1,22 @@
+import serial
+import ball_detection as bd
+
+
+class Encoding:
+    def __init__(self):
```

Comment:
You don't need to define an __init__ if you're not going to use it. But looks like there's some init needed? Either add in the needed code, or delete it if it's not needed.